packages/react/src/analytics/integrations/Guardian/Guardian.ts

```diff
@@ -0,0 +1,146 @@
+/**
+ * Guardian Integration. Loads the Guardian script and calls its methods.
```

Comment:
This description is not correct since this integration does not load a script as Vitorino once did.

Reply:
Updated